backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceHeaderImplBuilder.java

```diff
@@ -271,9 +271,15 @@ public void setC_BPartner_SalesRep_ID(final int sales_BPartner_ID)
 
 	public void setSalesRep_ID(final int salesRep_ID)
 	{
-		SalesRep_User_ID = checkOverrideID("SalesRep_ID", SalesRep_User_ID, salesRep_ID);
+		try
+		{
+			SalesRep_User_ID = checkOverrideID("SalesRep_ID", SalesRep_User_ID, salesRep_ID);
+		}
+		catch (final AdempiereException e)
```

Comment:
pls avoid using exception handling as a standard mechanism of flow control.
In your case, getting two different sales rep IDs it's a common case.
So, pls use if/else to handled that case and set the SalesRep_User_ID to -1


checkOverrideID

Reply:
done